src/main/java/com/synopsys/integration/jenkins/scan/service/ScannerArgumentService.java

```diff
@@ -17,23 +18,25 @@
 public class ScannerArgumentService {
     private static final String DATA_KEY = "data";
 
-    public List<String> getCommandLineArgs(Map<String, Object> scanParameters) {
+    public List<String> getCommandLineArgs(Map<String, Object> scanParameters, BitBucket bitBucket) {
         String stageName = getStageType(scanParameters);
         List<String> commandLineArgs = new ArrayList<>(getInitialBridgeArgs(stageName));
```

Comment:
it should be `getInitialBridgeArgs(BridgeParams.BLACKDUCK_STAGE)`

Reply:
Yes it is changed to `getInitialBridgeArgs(BridgeParams.BLACKDUCK_STAGE)`